bevrand.uitests/cypress/integration/Footer.feature

```diff
@@ -16,4 +16,3 @@ Feature: Footer
       | iphone-6   | profile                 |
       | ipad-2     | reroute/test            |
       | macbook-15 | playlistcreation/Asfasf |
-      | macbook-15 | playlistdetails         |
```

Comment:
Did these tests start failing, or are they no longer relevant?

Reply:
They are no longer working - since the playlistdetail page receives a playlist object and then makes a call to the highscore api. If you just go the link /playlistapi a call is made with 2 Undefined objects.

They should be readded at a later date